docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -0,0 +1,281 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="CustomConfig">
+  <title>Customizing Configurations</title>
+
+  <sect1 id="CustTechniques" role="nobrk">
+    <title>Customization Techniques</title>
+
+    <para>In this section, we will walk through creating a custom
+    configuration YAML file and deploying an HPCC
+    Systems<superscript>®</superscript> platform using the default
+    configuration plus the customizations.</para>
+
+    <para>There are several ways to customize a platform deployment. We
+    recommend using methods that allow you to best take advantage of the
+    configuration as code (CaC) practices. Configuration as code is the
+    standard of managing configuration files in a version control system or
+    repository.</para>
+
+    <para>The following is a list of common customization techniques:</para>
+
+    <itemizedlist>
+      <listitem>
+        <para>The first way to override a setting in the default configuration
+        on the command line using the <emphasis role="bold">--set</emphasis>
+        parameter. </para>
+
+        <para>This is the easiest, but the least compliant with CaC
+        guidelines. It is also harder to keep track of overrides this
+        way.</para>
+      </listitem>
+
+      <listitem>
+        <para>The second way is to download and modify the default YAML file
+        (..\helm\hpcc\values.yaml). This could comply with CaC guidelines if
+        you place that file under version control, but it makes it harder to
+        utilize a newer default configuration when one becomes
+        available.</para>
+      </listitem>
+
+      <listitem>
+        <para>The third way, is the one we typically use. Use the default
+        configuration plus a customization YAML file and use the -f parameter
+        to the helm command to use it. This uses the default configuration and
+        only overrides the settings specified in the customization
+        YAML..</para>
```

Comment:
bug: remove extra period

Reply:
I think it is worth noting at this point that multiple custom yaml files can be passed in in this way, e.g.
```
helm install myhpcc hpcc/hpcc -f myplanes.yaml -f mythors.yaml 
```